APITesters/SwiftAPITester/SwiftAPITester/PurchasesAPI.swift

```diff
@@ -152,6 +152,12 @@ private func checkIdentity(purchases: Purchases) {
     let loginComplete: (CustomerInfo?, Bool, Error?) -> Void = { _, _, _ in }
     purchases.logIn("", completion: loginComplete)
     purchases.logIn("") { _, _, _ in }
+    if #available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *) {
+        Task.init {
+            let (_, _): (CustomerInfo, Bool) = try await purchases.logIn("")
+            let _: CustomerInfo = try await purchases.logOut()
```

Comment:
another cool side-effect: we now get type-checks for the types returned. For completion blocks we can also get type checks but they're harder to do so we never did.

Reply:
That's really neat. I guess it's time I learn how to use `async` and  `await` 🙃

Reply:
I was like "wait but blocks are type safe too..."
I see you mean in the API testers! Yeah this is great.